README.md

```diff
@@ -1,7 +1,7 @@
 PELUX baseline manifests
 =========================
-This is the manifest repository for building the PELUX baseline for various
-hardware targets using Yocto
+This manifest repository is used for building the PELUX baseline for various
+hardware targets using the Yocto build tool.
```

Comment:
I don't feel it's more correct like this Yocto is usually referred to as more than the tool used for building. Maybe "using the Yocto project"?

Reply:
+1

Reply:
"using the Yocto project" is better. I'll change.

Reply:
Done.